packages/mds-provider-service/tests/service.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { uuid } from '@mds-core/mds-utils'
+import { MDSProviderServiceClient } from '../client'
+import { MDSProviderRepository } from '../repository'
+import { MDSProviderServiceManager } from '../service/manager'
+
+const TEST_PROVIDER_ID = uuid()
+
+describe('MDSProvider Repository Tests', () => {
+  beforeAll(async () => {
+    await MDSProviderRepository.initialize()
+  })
+
+  it('Run Migrations', async () => {
+    await MDSProviderRepository.runAllMigrations()
+  })
+
+  it('Revert Migrations', async () => {
+    await MDSProviderRepository.revertAllMigrations()
+  })
+
+  afterAll(async () => {
+    await MDSProviderRepository.shutdown()
+  })
+})
+
+const MDSProviderServer = MDSProviderServiceManager.controller()
+
+describe('MDSProvider Service Tests', () => {
+  beforeAll(async () => {
+    await MDSProviderServer.start()
+  })
+
+  it('Post MDSProvider', async () => {
+    const MDSProvider = await MDSProviderServiceClient.createMDSProvider({
+      provider_id: TEST_PROVIDER_ID,
+      provider_name: 'imatest',
+      color_code_hex: '#000'
+    })
+    expect(MDSProvider.provider_name).toEqual('imatest')
+  })
+
+  it('Get All MDSProviders', async () => {
+    const providers = await MDSProviderServiceClient.getMDSProviders()
+    expect(providers.length).toEqual(1)
+    const [provider] = providers
+    expect(provider?.provider_name).toEqual('imatest')
+  })
```

Comment:
A couple nits:

- Would probably be good to verify that the get-all test returns >1 entry (so we'd need to seed another one for the test)
- It may be nice to isolate the individual tests by nuking the repo after each `it`, at the moment these seem pretty order-dependent.